src/pywink/api.py

```diff
@@ -101,6 +101,23 @@ def get_sirens():
     return get_devices(device_types.SIREN)
 
 
+def get_subscription_key():
+    arequest_url = "{}/users/me/wink_devices".format(WinkApiInterface.BASE_URL)
+    response = requests.get(arequest_url, headers=API_HEADERS)
+    if response.status_code == 200:
+        response_dict = response.json()
+        first_device = response_dict.get('data')[0]
```

Comment:
@w1ll1am23 You could do something similar to `get_devices`, which offloads the work of actually processing the API response to `get_devices_from_response_dict`.

You could add `get_subscription_key_from_response_dict` which should be very testable.